invokeai/frontend/web/src/features/gallery/components/ImageGalleryContent.tsx

```diff
@@ -251,4 +253,5 @@ const ImageGalleryContent = () => {
   );
 };
 
+ImageGalleryContent.displayName = 'ImageGalleryContent';
```

Comment:
It ensures the component name always shows up in the build (visible by [React dev tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi)), when making test snapshots of JSX, etc-

I saw a few of the other components in this app so I added it, too - can remove if you prefer that

Some apps do it to make debugging easier, others don't to save KB in the final build size. I prefer to have it there since it doesn't add _**that much**_ to the build

I think since its very easy to emulate production on your local branches and I don't think I've seen JSX being snapshotted anywhere, its not a big impact either way for your app

Reply:
Yeah it's all good. I wasn't aware that it made a difference anywhere coz I thought the component name would always be inferred from the function or the export call. All good. Merged.

Reply:
Oh wait- just realized it may help users who happen to be React developers and bump into a bug they want to report - they could easily cite the component names, which would be handy

Reply:
Ah got it. Thanks for the explanation. Never faced this before.